src/DssSpell.t.sol

```diff
@@ -56,40 +56,42 @@ contract DssSpellTest is DssSpellTestBase {
         uint256 amount;
     }
 
-    function testPayments() private { // make public to enable
-        // uint256 prevSin = vat.sin(address(vow));
+    function testPayments() public { // make public to enable
```

Comment:
`testNewChainlogValues` and `testNewIlkRegistryValues` can be made private (and content can be commented).